core/templates/components/forms/schema-based-editors/schema-based-float-editor.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="schema-based-float-editor">
   <form #floatform="ngForm">
-    <span *ngIf="!expressionMode">
+    <span>
```

Comment:
expressionMode doesn't exist in the component.ts class. Here and everyother template related change mistakes caught by angular